pkgs/development/tools/language-servers/gopls/default.nix

```diff
@@ -2,17 +2,17 @@
 
 buildGoModule rec {
   pname = "gopls";
-  version = "0.13.0";
+  version = "0.13.1";
 
   src = fetchFromGitHub {
     owner = "golang";
     repo = "tools";
     rev = "gopls/v${version}";
-    sha256 = "sha256-E/QX3J/ux7GAG93b1K7OdDjYBAzte2HMa6bfALtXgcU=";
+    sha256 = "sha256-2eGnctA5HlNRGv9iV5HoT4ByA8fK/mTxldHll0UMD5c=";
   };
 
   modRoot = "gopls";
-  vendorSha256 = "sha256-e83y8bu0xKGEg7o2BWt4CzM2YosLDefFRgTfA0f3ZmI=";
+  vendorSha256 = "sha256-2H8Qh88ikmEqToGOCOoovcCh3dMToeFP/GavG9dlML8=";
```

Comment:
Prefer `vendorHash = ...` to `vendorSha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.

Reply:
whoops, this was not meant to be pushed to this PR